QUEST_LV_0200.tsv

```diff
@@ -98,10 +98,10 @@ QUEST_LV_0200_20150317_000097	The sacrifice will continue unless someone stops t
 QUEST_LV_0200_20150317_000098	Joint Investigation Team Fray
 QUEST_LV_0200_20150317_000099	I'm Fray.{nl}Listening to the story, the situation sounds bad.{nl}If the information you and Scott researched is correct.
 QUEST_LV_0200_20150317_000100	{nl}Then, I will have to do something.[nl}Please defeat some Red Vubbes first. They have been bothering me lately.
-QUEST_LV_0200_20150317_000101	Please, I ask of you. .{nl}I get sick just looking at the red color.
+QUEST_LV_0200_20150317_000101	$Please, I ask to you. .{nl}I get sick just looking at the red color.
```

Comment:
@darkhershel Wrong; "Ask to" is only correct when you have something in between: you can ask SOMETHING to someone. Or if the to is conjugating a verb; Ask to go to the restroom.
"Ask of" is used as a phrase on its own; Ask of you to do something. "May I ask a favour of you?"